keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/titus/verification/TestContainerVerificationEvaluator.kt

```diff
@@ -64,22 +64,26 @@ class TestContainerVerificationEvaluator(
 
     return runBlocking {
       withContext(IO) {
+        val containerJobConfig = ContainerJobConfig(
+          application = verification.application ?: context.deliveryConfig.application,
+          location = verification.location,
+          credentials = verification.location.account,
+          image = verification.image,
+
+          // These are deprecated and will eventually go away, since users now pass the info in the image field
+          repository = verification.repository,
```

Comment:
It seems to me that, by the time we get here, we could already determine the image from the different optional fields, and so `ContainerJobConfig` should only have an `image` field. The only place we should keep the old fields for backwards compatibility is in the `TestContainerVerification` class.

Reply:
That's a good idea. I'm going to move the imageId property to TestContainerVerification.